tests/conftest.py

```diff
@@ -18,6 +18,8 @@
 from dj.config import Settings
 from dj.utils import get_project_repository, get_session, get_settings
 
+from .sql.parsing.queries import *  # pylint: disable=W0401,W0614
```

Comment:
Imo we should list out all of the imports explicitly here and accept the overhead of having to update it as we add more queries. It makes it easier to figure out what's happening when trying to understand the query fixtures for the first time.